src/drivers/intel/cavs/hda-dma.c

```diff
@@ -50,7 +50,8 @@
 #define DGCS_GEN	BIT(26)
 #define DGCS_FWCB	BIT(23)
 #define DGCS_BSC	BIT(11)
-#define DGCS_BOR	BIT(10) /* buffer overrun */
+#define DGCS_BOR	BIT(10) /* buffer overrun (input streams) */
+#define DGCS_BUR	BIT(10) /* buffer underrun (output streams) */
```

Comment:
Should both be bit 10 ?

Reply:
@lgirdwood Yes, but have different names depends on the stream's direction. It's just cosmetic change.

Reply:
ok, can you comment this otherwise it may confuse.

Reply:
@lgirdwood Added additional comment.